newproject01/newproject01/q2.c

```diff
@@ -0,0 +1,39 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#include <string.h>
+#define N 51
+char letterAtIndex(char*, int);
+main2()
+{
+	char string[N], returnChar;
+	int m;
+
+	printf("Enter string(not more than 50 chars), please: ");
+	if (scanf("%s", string) != 1)
+		return 1;
+
+	printf("Enter any index in the string, please: ");
+	if (scanf("%d", &m) != 1)
+		return 1;
+	returnChar = letterAtIndex(string, m);
+	printf("The index char in organized string is: %c", returnChar);
+	return 0;
+}
+char letterAtIndex(char* string, int index)
```

Comment:
The request in the exercise was that the parameters will be `char* s, unsigned int m`.